content/contributing/membership/other/contents.lr

```diff
@@ -0,0 +1,11 @@
+title: Other
+---
+_discoverable: yes
+---
+body: If you'd like to talk to someone about other branding or sponsorship opportunities, or you'd like other payment options for your membership, please direct enquiries to `Russell Keith-Magee <mailto:russell@pybee.org>`__.
+---
+display: yes
+---
+summary: For individuals
+---
+cost: 5000.2
```

Comment:
I found some members had `other` as a membership. Can we assume Other == Ad Hoc, or is there a difference there?